chrome/uninstall.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="uninstall.js"></script>
+    <title></title>
+  </head>
+  <body>
+    <button id="uninstall">Uninstall</button>
```

Comment:
Sorry, I meant this just as a placeholder. I wanted to wait with i18n boilerplate code until we know which strings exactly we want to have. Nevertheless, I added it now :)